OutOfSchool/OutOfSchool.BusinessLogic/Services/WorkshopDrafts/WorkshopDraftService.cs

```diff
@@ -150,7 +150,7 @@ await workshopDraftRepository.SaveChangesAsync()
             .ConfigureAwait(false);
 
         var createdDraftDto = mapper.Map<WorkshopDraftResponseDto>(createdDraftWithAssociatedTeachers);
-        createdDraftDto.Tags = mapper.Map<List<TagDto>>(tags);
+        // createdDraftDto.Tags = mapper.Map<List<TagDto>>(tags);
```

Comment:
please don't leave commented code. If it is needed - uncomment, if it is not needed - delete.